src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -31,6 +31,8 @@ This file is part of the DITA Open Toolkit project.
 See the accompanying LICENSE file for applicable license.
 -->
 
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
```

Comment:
Another comment to remove - it doesn't look like this file is touched apart from the comment.

I'll skip commenting on the same comment for other XSL files.